lib/Core/Search/Elasticsearch/DocumentMapper/BlockFieldMapper.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace Netgen\IbexaSearchExtra\Core\Search\Elasticsearch\DocumentMapper;
+
+use Ibexa\Contracts\Core\Persistence\Content as SPIContent;
+
+abstract class BlockFieldMapper
+{
+    /**
+     * Indicates if the mapper accepts the given $content for mapping.
+     */
+    abstract public function accept(SPIContent $content): bool;
+
+    /**
+     * Maps given $content to an array of search fields.
+     *
+     * @return \Ibexa\Contracts\Core\Search\Field[]
+     */
+    abstract public function mapFields(SPIContent $content): array;
+}
```

Comment:
Missing new line at the end of file, you can configure that is PHPStorm to be added automatically.

Reply:
resolved in 6dc18e42f913073475c43a7b50a90abfc28e727f